### ISSUE TITLE
Is it still not possible to use GPU-based media transcoding? 

### ISSUE THREAD

**Issue by mkanet** [cjmurph/PmsService#58]
I was interested in looking for a solution that would automatically restart Plex Media Server if it crashes.  However, I just realized that I might not be able to do GPU based transcoding anymore if the executable runs in session 0.

If there is a GPU restriction to running as a Windows Service, If may be better for me to just make a PowerShell script that checks if Plex Media Server.exe is in memory; if not, to launch it again.  That way, everything will work without any weird limitations.

**Comment by SubNoizey** [comment]
> I was interested in looking for a solution that would automatically restart Plex Media Server if it crashes. However, I just realized that I might not be able to do GPU based transcoding anymore if the executable runs in session 0.
> 
> If there is a GPU restriction to running as a Windows Service, If may be better for me to just make a PowerShell script that checks if Plex Media Server.exe is in memory; if not, to launch it again. That way, everything will work without any weird limitations.

Did you end up making something that fixed this? I'm in a similar boat

**Comment by mkanet** [comment]
@SubNoizey yes I did.  However, it's not an elegant solution (however, it does work).  Instead of trying to execute the script as a Windows Service, I am executing the script (see script below) via the Windows Task Scheduler.  The script will need to start automatically _after the PC logs into the Windows Desktop (using the same Windows user account that's logged into the Windows Desktop_.  _The user also needs to be a Windows Desktop Administrator (part of the local Windows "Administrators" group._  

The script is a simple one-liner PowerShell command:

```
if (!(Get-Process -Name "Plex Media Server" -ErrorAction SilentlyContinue)) {Start-Process -FilePath "C:\Program Files (x86)\Plex\Plex Media Server\Plex Media Server.exe"}
```

I also converted that script into a Windows Executable that's hidden using [PS2EXE](https://www.powershellgallery.com/packages/ps2exe) .. so when the script is started, it will be completely invisible.
